package.json

```diff
@@ -326,16 +325,13 @@
     "mock-fs": "^4.4.2",
     "murmurhash3js": "3.0.1",
     "mutation-observer": "^1.0.3",
-    "ncp": "2.0.0",
     "nock": "8.0.0",
     "node-sass": "^4.9.0",
     "pixelmatch": "4.0.2",
     "prettier": "^1.14.0",
     "proxyquire": "1.7.11",
     "simple-git": "1.37.0",
     "sinon": "^5.0.7",
-    "source-map": "0.5.6",
```

Comment:
I believe this was here at some point to pin the version of source-map, but doesn't look like that is still necessary.